config/initializers/assets.rb

```diff
@@ -12,4 +8,8 @@
 
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
-Rails.application.config.assets.precompile += %w( search.js )
+# addition of '.svg' allows custom logo to display
+Rails.application.config.assets.precompile += %w( search.js '.svg')
+
+# css_compressor needed for svg to display according to: https://stackoverflow.com/a/36992492
+Rails.application.config.assets.css_compressor = :sass
```

Comment:
This is just a more sensible reorganization of the lines I had before, and should have no functional impact.